### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by iazehaf**

### CODE REVIEW

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml [hunk 1]
```diff
@@ -53,7 +52,7 @@ dash-mantine-components = "^0.12.1"
 dash-uploader = "^0.6"
 {% elif cookiecutter.dash_ui == "awc" %}
 ansys-web-components-dash = {version = ">=0.0.dev4", source = "solutions-private-pypi"}
```
Comment (ansRMorin) [inline src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml:54]: should we remove this package?

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml [hunk 2]
```diff
@@ -53,7 +52,7 @@ dash-mantine-components = "^0.12.1"
...
-ansys-solutions-dash-super-components = {version = "^0.1.dev0", python = ">=3.9,<3.11"}
+ansys-solutions-dash-super-components = {version = "0.1.dev0", source = "solutions-private-pypi", python = ">=3.10,<3.11"}
```
Comment (ansRMorin) [inline src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml:55]: this is a duplicate. I guess we should remove this!

**Review by ansRMorin — APPROVED** [review]
Looks good to me! I would just make those small changes to the toml file and then I think it's good to go.

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml (hunk 1 above)
Comment (iazehaf) [reply src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml:54]: This package is needed for the AWC UI model

File: src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml (hunk 2 above)
Comment (iazehaf) [reply src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml:55]: This is not a duplicate but the config for the AWC mode.

**Pull request merged**